### ISSUE TITLE
Unable to create a new permission role, stations goes off screen

### ISSUE THREAD

**Issue by SC2Mitch** [AzuraCast/AzuraCast#4666]
### Installation Method

Docker Installation

### AzuraCast Release Channel

Rolling Release Channel

### What happened?

OK so if you have more than 10+ stations you're unable to create a new role because the is off the screen. 

You must zoom out of the page to see the rest of the stations which isn't user-friendly at all. 

Gif of the issue: https://puu.sh/Ihd1e/df113f9bee.gif
